src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -76,7 +76,7 @@ function(mode, isCurve, isBezier,isQuadratic, isContour, shapeKind){
     this.immediateMode.vertexPositions,
     this.immediateMode.vertexColors);
   if(mode){
-    if(this.drawMode === 'fill'){
+    if(this.drawMode === 'fill' || 'texture'){
```

Comment:
does this work? I think this would get evaluated as 
`if ( (this.drawMode === 'fill') || 'texture')` which would always result in "true", since the second part of the OR is a non-falsey value. I could be wrong..
rewriting as:
`if ( this.drawMode === 'fill' || this.drawMode === 'texture')` removes any ambiguity.